internal/manifests/collector/poddisruptionbudget.go

```diff
@@ -18,37 +18,45 @@ import (
 	policyV1 "k8s.io/api/policy/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"github.com/open-telemetry/opentelemetry-operator/internal/api/convert"
 	"github.com/open-telemetry/opentelemetry-operator/internal/manifests"
 	"github.com/open-telemetry/opentelemetry-operator/internal/manifests/manifestutils"
 	"github.com/open-telemetry/opentelemetry-operator/internal/naming"
 )
 
-func PodDisruptionBudget(params manifests.Params) *policyV1.PodDisruptionBudget {
+func PodDisruptionBudget(params manifests.Params) (*policyV1.PodDisruptionBudget, error) {
+	otelCol, err := convert.V1Alpha1to2(params.OtelCol)
```

Comment:
can you open an issue for cleaning this up after this is merged?

Reply:
- https://github.com/open-telemetry/opentelemetry-operator/issues/2592